src/modules/comments/comments.service.ts

```diff
@@ -0,0 +1,103 @@
+import { NotFoundException } from './../../shared/exceptions/http.exception';
+import { Repository } from 'typeorm';
+import {
+  BadRequestException,
+  InternalServerErrorException,
+} from '../../shared/exceptions/http.exception';
+import { Comment } from './comments.entity';
+import { CreateCommentDto } from './dto/create-comment.dto';
+import { UpdateCommentDto } from './dto/update-comment.dto';
+import { v4 as uuidv4 } from 'uuid';
+import { User } from '../user/user.entity';
+import { Blog } from '../blog/blog.entity';
+
+export class CommentService {
+  constructor(
+    private commentRepository: Repository<Comment>,
+    private userRepository: Repository<User>,
+    private blogRepository: Repository<Blog>
+  ) {}
+
+  async createComment(blogId:string, createCommentDto: CreateCommentDto) {
```

Comment:
I didn't see usage of this service in controller.

Reply:
![Screenshot_1](https://github.com/user-attachments/assets/46477bda-d3f0-4f8c-8dd5-7a52228916ce)

I just thought that the comment should be created for some blog, so it should be in the blog controller.
Should I move the this controllers to comments controller?

Reply:
I put them there because in the documentation of our assignment, these services are used in the route "blogs/id/comments". Should I replace them? I just thought they were supposed to be there.